src/main/scala/temple/generate/server/go/common/GoCommonGenerator.scala

```diff
@@ -42,4 +42,43 @@ object GoCommonGenerator {
         ),
       ),
     )
+
+  /** Generate declaration and assignment statement */
+  private[go] def genDeclareAndAssign(expr: String, identifiers: String*): String =
```

Comment:
I see why you've had to do this, but I wish you could switch the order of the args 😢

Reply:
Yeah it makes me sad 😞

Reply:
Could curry it and reverse them, but up to you if you prefer that